packages/api/models/schemas.js

```diff
@@ -360,7 +360,7 @@ module.exports.provider = {
     protocol: {
       title: 'Protocol',
       type: 'string',
-      enum: ['http', 'ftp', 'sftp', 's3'],
+      enum: ['http', 'https', 'ftp', 'sftp', 's3'],
```

Comment:
We should also update the all places where we use the selector function for this to work and not fail:

e.g. https://github.com/cumulus-nasa/cumulus/blob/master/packages/ingest/granule.js#L454

Reply:
updated all the placed I can find.  Currently the DiscoverGranules doesn't queue granule, we may want to add steps to perform end2end https ingest.
https://github.com/cumulus-nasa/cumulus/blob/master/example/workflows.yml#L40